service/service.go

```diff
@@ -272,6 +303,11 @@ func New(config Config) (*Service, error) {
 
 func (s *Service) Boot(ctx context.Context) {
 	s.bootOnce.Do(func() {
+		err := s.routeTableService.Boot(ctx)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
I guess the panic is unavoidable here but Mr @kopiczko had ideas about changing the error handling in the boot process.

Reply:
I think it can be avoided in this case.

Reply:
I removed boot anyway.